README.md

```diff
@@ -6,7 +6,13 @@ release](https://img.shields.io/github/release/SpicyPizza/create-envfile.svg?sty
 marketplace](https://img.shields.io/badge/marketplace-create--env--file-blue?logo=github&style=flat-square)](https://github.com/marketplace/actions/create-env-file)
 [![Licence](https://img.shields.io/github/license/SpicyPizza/create-envfile)](https://github.com/SpicyPizza/create-envfile/blob/master/LICENSE)
 
-### SpicyPizza/create-envfile@v1.2
+## Versions
+
+Please use this version of your action config file:
+
+* `SpicyPizza/create-envfile@master`
+
+* `SpicyPizza/create-envfile@v1.2`
```

Comment:
I'm not sure this section makes sense. Do you mean to suggest that people should choose one of them to use?

Reply:
Maybe, just to show the current versions.

Reply:
I'll add a commit to make it more verbose :+1: